### DISCUSSION TITLE
SMTP servers and email addresses?

### DISCUSSION THREAD

**Discussion by ackledotdev** [is-a-dev/register#9243] [Q&A] [answered]
I saw this in the domain management dashboard and was wondering what it is. Do you host your own SMTP servers? If so, how can I use them to create an email address with my subdomain and use it with an email client?

Thanks.

![Screenshot 2023-11-07 3 37 25 PM](https://github.com/is-a-dev/register/assets/111009970/724ca5da-5ee2-4f00-8c3e-a9121061eb17)

**Comment by devashish2024** [comment]
is-a.dev does not provide SMTP, its likely your hosting's feature, but if any issue occurs, its because we don't provide. Mark this as answer if it helped you!

**Reply by iostpa** [reply, marked as answer]
we actually have SMTP but its currently visible to beta testers in the discord server

**Reply by devashish2024** [reply]
Oh damn
